src/pages/Login.tsx

```diff
@@ -31,7 +32,10 @@ export const Login: React.FC<RouteComponentProps> = ({ location }) => {
   if (isAuthenticated) {
     return <Redirect to={getReferrer(location)} />;
   }
-
+  if (isProduction()) {
+    window.location.href = `${getUiUrl()}/login`;
```

Comment:
Any Reason to not just return a `<Redirect/>` over here?

Reply:
because spruce login is still needed in test environment

Reply:
setting `window.location.href` redirects the page why not just use the built in `react-router` `<Redirect /> ` component? Since they do the same thing.

Reply:
when we are in the test environment the redirect wont happen because of the isProduction conditional and we will still use the spruce login.

Reply:
> Since they do the same thing.

☝️ I guess that's why. I didn't consider it because I guess I view view react-router as in-app routing. Is there any benefit to using the react router Redirect?

Reply:
setting window.location.href causes the browser to refresh on the redirect while using history.push() or `<Redirect/>` dont . 
https://stackoverflow.com/questions/51713513/window-location-href-vs-redirect-utility-of-react-router-dom

Reply:
well i guess if this redirects out to an external page it will refresh anyways.